src/encoded/static/components/encyclopedia.js

```diff
@@ -1,49 +1,492 @@
+                                                actuator={
+                                                    <button className="download-icon-button" type="button">
+                                                        {svgIcon('download')}
+                                                    </button>
+                                                }
+                                                modalContent={
+                                                    <div className="download-selections">
+                                                        <div className="encyclopedia-subhead">Download cell-type agnostic cCREs:</div>
+                                                        {fileOptions.map((download) => <label className="download-checkbox"><input type="checkbox" name="checkbox" value={download} onChange={() => changedownloadHref(download)} checked={(selectedFiles.indexOf(download) > -1)} />{download}</label>)}
+                                                    </div>
+                                                }
+                                            />
+                                            <span>Download cell-type agnostic cCREs</span>
+                                        </div>
+                                    : null}
+                                    {browserController ?
+                                        <div className="download-line">
+                                            <BatchDownloadActuator
+                                                controller={browserController}
+                                                actuator={
+                                                    <button className="download-icon-button" type="button">
+                                                        {svgIcon('download')}
+                                                    </button>
+                                                }
+                                            />
+                                            <span>Download selected annotations</span>
+                                        </div>
+                                    : null}
+                                    <hr />
+                                </div>
+                                <div className="series-wrapper">
+                                    <div>There are {vizFiles.length} file{vizFiles.length === 1 ? '' : 's'} displayed out of {totalFiles} file{totalFiles.length > 1 ? '' : 's'} that match the selected filters.</div>
+                                    <Panel>
+                                        <div className="file-gallery-container encyclopedia-file-gallery">
+                                            {(facetData && facetData.facets && facetData.facets.length > 0) ?
+                                                <div className="file-gallery-facet-redirect" onClick={(e) => handleFacetClick(e)}>{/* eslint-disable-line */}
```

Comment:
Why are you disabling eslint on this line? I see no reason to.

Reply:
There's an error here. For a11y compliance, click events need to have a role and keyboard listeners. However, this isn't a real button, it's a redirect for the buttons/links in the facet wrapper, so it's not a real button and I don't think we want it to be a thing people actually click on, if that makes sense. Maybe I'm wrong.